utils/scripts/util.py

```diff
@@ -106,3 +108,22 @@ def render_template(search_path, template_file, output_file, replace_vars):
         print("Failed to render template and create json "
               "file {}".format(output_file))
         sys.exit(1)
+
+def read_data_from_json(filename=None):
+    """
+    Helper to read Json file
+    """
+    try:
+        with open(filename, 'r') as f:
+            data = json.load(f)
```

Comment:
use safe_load option

Reply:
hi arthy it was  not working with safe load